# Request
int32 drum_id
---
# Response
int32 drum_pos
